pkg/controller/service/testdata/ingress/basic/expected.golden

```diff
@@ -179,7 +202,7 @@ status:
     type: defined
   endpoints:
   - address: oneimage-app-name-a5b0aade.local.oss-acorn.io
-    publishProtocol: https
```

Comment:
I think this change is also expected. Previously, it will set `publishProtocol` to https if one ingress TLS is found, which is not correct. After this change, if custom domain is setting ingress but cluster domain doesn't have cert provisioned, the cluster domain remain to be http.

https://github.com/acorn-io/acorn/blob/main/pkg/publish/ingress.go#L206-L209